bool sky_light
chunk_meta_msg[] metadata
uint8[] data
